package.json

```diff
@@ -4,14 +4,19 @@
   "private": true,
   "scripts": {
     "test": "cross-env NODE_ENV=test nyc mocha --file test/test.js --timeout 6666 --require babel-polyfill --require babel-register --exit && nyc report --reporter=html --reporter=text-lcov | coveralls",
-    "server": "cross-env NODE_ENV=development nodemon --exec babel-node server/server.js",
-    "build-server": "cross-env NODE_ENV=production babel -d ./build ./server -s"
+    "server": "cross-env NODE_ENV=development nodemon --exec babel-node src/server.js",
+    "build-server": "cross-env NODE_ENV=production babel -d ./build ./src -s",
+    "start": "./node_modules/nodemon/bin/nodemon.js src/server.js"
```

Comment:
`npm run server` already starts the server though?

Reply:
Typing `npm start` is faster than `npm run server` :P